doc/whats_new.rst

```diff
@@ -378,6 +378,11 @@ Bug fixes
    - Fix bug where stratified CV splitters did not work with
      :class:`linear_model.LassoCV`. :issue:`8973` by `Paulo Haddad <paulochf>`.
 
+   - Fix :class:`semi_supervised.LabelPropagation` to always do hard clamping.
+     Its ``alpha`` parameter now defaults to 0 and the parameter is deprecated
```

Comment:
Nvm, I just read the documentation below. Can you please change the [narrative documentation](http://scikit-learn.org/stable/modules/label_propagation.html) so that there is no confusion.